File: examples/always-high.cpp
#include<iostream>
#include <vector>

int main(){
    // read n, k, w and j
    int n, k, w, j;
    std::cin >> n >> k >> w >> j;

    // repeating for the whole game
    while (true){
        // outputting the highest number
        std::cout << k << std::endl;
        // reading the input but ignoring it
        std::vector<int> submitted_numbers(n);
        for (int i = 0; i < n; i++) {
            std::cin >> submitted_numbers[i];
        }
    }
}

File: examples/always-high.py
# read n, k, w and j
n, k, w, j = map(int, input().split())

# repeating for the whole game
while True:
    # outputting the highest allowed number
    print(k)
    # reading the input but ignoring it
    submitted_numbers = list(map(int, input().split()))